apps/dot_ext/admin.py

```diff
@@ -76,47 +76,41 @@ def clean_agree(self):
         return self.cleaned_data.get('agree')
 
 
+@admin.register(MyApplication)
 class MyApplicationAdmin(admin.ModelAdmin):
     form = CustomAdminApplicationForm
     list_display = ("name", "user", "authorization_grant_type", "client_id",
                     "require_demographic_scopes", "scopes",
                     "created", "updated", "active", "skip_authorization")
-    list_filter = ("name", "user", "client_type", "authorization_grant_type",
+    list_filter = ("client_type", "authorization_grant_type",
                    "require_demographic_scopes", "active", "skip_authorization")
     radio_fields = {
         "client_type": admin.HORIZONTAL,
         "authorization_grant_type": admin.VERTICAL,
     }
-    raw_id_fields = ("user", )
 
+    search_fields = ('name', 'user__username', '=client_id', '=require_demographic_scopes', '=authorization_grant_type')
 
-admin.site.register(MyApplication, MyApplicationAdmin)
+    raw_id_fields = ("user", )
 
 
+@admin.register(MyAccessToken)
 class MyAccessTokenAdmin(admin.ModelAdmin):
     list_display = ('user', 'application', 'expires', 'scope')
     search_fields = ('user__username', 'application__name',)
```

Comment:
The following would be useful to add to the search fields:

```suggestion
    search_fields = ('user__username', 'application__name', 'token', 'source_refresh_token__token')
```

Reply:
made token and source_refresh_token searchable